modules/build/src/main/scala/scala/build/options/ScalaNativeOptions.scala

```diff
@@ -68,9 +70,18 @@ final case class ScalaNativeOptions(
 
   def platformSuffix: String =
     "native" + ScalaVersion.nativeBinary(finalVersion).getOrElse(finalVersion)
-  def nativeDependencies: Seq[AnyDependency] =
-    Seq("nativelib", "javalib", "auxlib", "scalalib")
-      .map(name => dep"org.scala-native::$name::$finalVersion")
+
+  private def nativeScalalibDependency(scalaVersion: String): AnyDependency =
+    if (scalaVersion.startsWith("2."))
+      dep"org.scala-native::scalalib::$finalVersion"
+    else
+      dep"org.scala-native::scala3lib::$finalVersion"
+
+  def nativeDependencies(scalaVersion: String): Seq[AnyDependency] =
+    Seq("nativelib", "javalib", "auxlib")
```

Comment:
In theory if scala-cli would also handle all the transitive dependencies on it's own we can specify only `scalalib` or `scala3lib` (when using Scala3), all other libraries like nativelib, javalib, auxlib, and not listed here clib, posixlib and windowslib are dependencies of scalalib/scala3lib